### DISCUSSION TITLE
How to remap the vi_normal `i` keybinding

### DISCUSSION THREAD

**Discussion by SpacewaIker** [nushell/nushell#13452] [Q&A]
I use colemak, so in vim I remapped the `i` to `l` to go to insert mode. I'd like to do the same in nushell for the vi mode, but I can't find how to do this. I've currently remapped the `hjkl` to `mnei` for cursor movement, but I can't remap `l` to go to vi_insert mode from vi_normal mode.

How can I achieve this?

Thanks!

**Comment by fdncred** [comment]
You may be able to get somewhere with the keybindings section of the config.nu file. That's where I'd start.

**Reply by SpacewaIker** [reply]
Yes I assumed so, my question was more what I should put as the `event` for the keybinding configuration

**Reply by fdncred** [reply]
`keybindings list` (if it's working) should show you options. It's not working for me right now. I'm not sure why. You can also see what is mapped as default with `keybindings default`. Some of the vi keybindings are not changeable. I'm not sure if this is one or not without list working.

**Reply by SpacewaIker** [reply]
`keybindings list` doesn't show anything for me either. As for `keybindings default`, I don't see a mapping for going to insert mode from normal mode, so perhaps this keybinding cannot be changed as of now...

**Reply by fdncred** [reply]
I figured out that there is a bug in `keybinding list` however you if you do `keybindings list --help` you'll see that you can specifiy additional flags that will show all the data for each flag.

**Reply by fdncred** [reply]
There's a PR to fix keybindings list now.
